RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -77,13 +77,37 @@ struct PurchaseInformation {
     /// Note: `nil` for non-subscriptions, and for expiring subscriptions
      let renewalDate: Date?
 
+    /// The date an unsubscribe was detected.
+    ///
+    /// Note: `nil` for non-subscriptions, and for expiring subscriptions
+    let unsubscribeDetectedAt: Date?
+
+    /// Date when RevenueCat detected any billing issues with this subscription.
+    ///
+    /// Note: `nil` for non-subscriptions, and for expiring subscriptions
+    let billingIssuesDetectedAt: Date?
+
+    /// Date when any grace period for this subscription expires/expired.
+    /// nil if the customer has never been in a grace period.
+    ///
+    /// Note: `nil` for non-subscriptions, and for expiring subscriptions
+    let gracePeriodExpiresDate: Date?
+
+    let refundedAtDate: Date?
```

Comment:
I guess this is not public so it doesn't matter but any reason this has no docs when the others do?

Reply:
https://github.com/RevenueCat/purchases-ios/pull/5184/commits/91338eabdf190ab04f52564e7f84bba57cc5e0af